### PR TITLE
Create "Subscribe to Astro Release Notes" page

### LINKED ISSUE #1667: New Doc for Subscribing to Astro OSS Release Notes
### Context

We added a new **RSS** feed button to Astro release notes this week. This has been an eagerly awaited feature that will allow customers to subscribe and get notifications about Astro release notes in their browser or in slack.

image

### Ask

A lot of folks know how to use RSS feeds, but the customer team requested a URL with instructions.

- Create a new doc in the **Release Notes** menu called `Subscribe to Astro Release Notes`
- Use this as starting template: https://docs.google.com/document/d/1a0PqwpZzyyJqfsNRDYcmyTEZhQnvcnoDwnXfihxNEl4/edit?usp=sharing

cc: @katiebrady-astro @juliana-astro

### PR DESCRIPTION
**Pull request by katiebrady-astro**
Closes #1667 

image

### CODE REVIEW

File: astro/release-notes-subscribe.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+sidebar_label: 'Subscribe to Astro Release Notes'
+title: 'Subscribe to Astro Release Notes'
+id: release-notes-subscribe
+description: You can subscribe to the RSS feeds directly, or configure updates to be sent directly to your email or Slack.
```
Comment (kwcanuck) [inline astro/release-notes-subscribe.md:5]:
```suggestion
description: Subscribe to Astro Release Notes RSS feeds, or have updates sent to your email or Slack.
```

File: astro/release-notes-subscribe.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+---
+
+**We are excited to announce that you can now subscribe to Astronomer Release Notes!**
```
Comment (kwcanuck) [inline astro/release-notes-subscribe.md:8]:
```suggestion

```

File: astro/release-notes-subscribe.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+Astronomer has created RSS feeds for Astro Release Notes, Astro CLI Release Notes, and Astro Runtime Release Notes. You can subscribe to the RSS feeds directly, or configure updates to be sent directly to your email or Slack.
```
Comment (kwcanuck) [inline astro/release-notes-subscribe.md:9]:
```suggestion
You can subscribe to RSS feeds for Astro Release Notes, Astro CLI Release Notes, and Astro Runtime Release Notes. You can receive RSS feeds directly, or updates can be sent to your email or Slack account.
```

File: astro/release-notes-subscribe.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+## Get Updates to your Email Inbox
```
Comment (kwcanuck) [inline astro/release-notes-subscribe.md:11]:
```suggestion
## Subscribe to email updates
```

File: astro/release-notes-subscribe.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+With an RSS feed reader extension, such as Chrome’s free extension [Feeder](https://chrome.google.com/webstore/detail/rss-feed-reader/pnjaodmkngahhkoihejjehlcdlnohgmp?hl=en), you can subscribe to get updates from a feed sent directly to your email inbox.
```
Comment (kwcanuck) [inline astro/release-notes-subscribe.md:13]:
```suggestion
To have RSS feed updates sent to your email, use a browser RSS feed reader extension. For example, the [Chrome RSS Feed Reader](https://chrome.google.com/webstore/detail/rss-feed-reader/pnjaodmkngahhkoihejjehlcdlnohgmp?hl=en).
```

File: astro/release-notes-subscribe.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+_Note: Safari does not natively support viewing or configuring RSS feeds on Desktop or Mobile._
```
Comment (kwcanuck) [inline astro/release-notes-subscribe.md:15]:
```suggestion
:::info

You can't view or configure RSS feeds on desktop or mobile devices that use Safari.

:::
```

File: astro/release-notes-subscribe.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+## Get Updates on Slack
```
Comment (kwcanuck) [inline astro/release-notes-subscribe.md:17]:
```suggestion
## Subscribe to Slack updates
```

File: astro/release-notes-subscribe.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+Add the **Astro Release Notes** feed to a Slack Channel with the command
```
Comment (kwcanuck) [inline astro/release-notes-subscribe.md:19]:
```suggestion
In Slack, run the following slash command to add the Astro Release Notes feed to a Slack Channel:  
```

File: astro/release-notes-subscribe.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+## Get Updates to your Email Inbox
+
+With an RSS feed reader extension, such as Chrome’s free extension [Feeder](https://chrome.google.com/webstore/detail/rss-feed-reader/pnjaodmkngahhkoihejjehlcdlnohgmp?hl=en), you can subscribe to get updates from a feed sent directly to your email inbox.
+
+_Note: Safari does not natively support viewing or configuring RSS feeds on Desktop or Mobile._
+
+## Get Updates on Slack
+
+Add the **Astro Release Notes** feed to a Slack Channel with the command  
+/feed subscribe [https://docs.astronomer.io/astro-release-notes.xml](https://docs.astronomer.io/astro-release-notes.xml)
```
Comment (kwcanuck) [inline astro/release-notes-subscribe.md:20]:
```suggestion
`/feed subscribe [https://docs.astronomer.io/astro-release-notes.xml](https://docs.astronomer.io/astro-release-notes.xml)`
```

File: astro/release-notes-subscribe.md [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+
+Add the **Astro Runtime Release Notes** feed to a Slack Channel with the command  
+/feed subscribe [https://docs.astronomer.io/astro-runtime-release-notes.xml](https://docs.astronomer.io/astro-runtime-release-notes.xml)
```
Comment (kwcanuck) [inline astro/release-notes-subscribe.md:23]:
```suggestion
`/feed subscribe [https://docs.astronomer.io/astro-runtime-release-notes.xml](https://docs.astronomer.io/astro-runtime-release-notes.xml)`
```

File: astro/release-notes-subscribe.md [hunk 11]
```diff
@@ -0,0 +1,28 @@
...
+
+Add the **Astro CLI Release Notes** feed to a Slack Channel with the command
```
Comment (kwcanuck) [inline astro/release-notes-subscribe.md:25]:
```suggestion
Run the following slash command to add the Astro CLI Release Notes feed to a Slack channel:  
```

File: astro/release-notes-subscribe.md [hunk 12]
```diff
@@ -0,0 +1,28 @@
...
+## Get Updates on Slack
+
+Add the **Astro Release Notes** feed to a Slack Channel with the command  
+/feed subscribe [https://docs.astronomer.io/astro-release-notes.xml](https://docs.astronomer.io/astro-release-notes.xml)
+
+Add the **Astro Runtime Release Notes** feed to a Slack Channel with the command  
+/feed subscribe [https://docs.astronomer.io/astro-runtime-release-notes.xml](https://docs.astronomer.io/astro-runtime-release-notes.xml)
+
+Add the **Astro CLI Release Notes** feed to a Slack Channel with the command  
+/feed subscribe [https://docs.astronomer.io/astro-cli-release-notes.xml](https://docs.astronomer.io/astro-cli-release-notes.xml)
```
Comment (kwcanuck) [inline astro/release-notes-subscribe.md:26]:
```suggestion
`/feed subscribe [https://docs.astronomer.io/astro-cli-release-notes.xml](https://docs.astronomer.io/astro-cli-release-notes.xml)`
```

**Review by kwcanuck — APPROVED** [review]
Looks good. Should instructions be provided for configuring an RSS feed, or is it too obvious?

File: astro/release-notes-subscribe.md [hunk 13]
```diff
@@ -0,0 +1,28 @@
...
+
```
Comment (jwitz) [inline astro/release-notes-subscribe.md:27]:
```suggestion

- Astro product release notes: `/feed subscribe https://docs.astronomer.io/astro-release-notes.xml`

- Astro Runtime release notes: `/feed subscribe https://docs.astronomer.io/astro-runtime-release-notes.xml`

- Astro CLI release notes:  `/feed subscribe https://docs.astronomer.io/astro-cli-release-notes.xml`

```

File: astro/release-notes-subscribe.md [hunk 14]
```diff
@@ -0,0 +1,28 @@
...
+_Note: you may need Team Owner access on Slack to configure a public channel. You can read more about the Slack RSS feed integration [here](https://slack.com/help/articles/218688467-Add-RSS-feeds-to-Slack)._
```
Comment (jwitz) [inline astro/release-notes-subscribe.md:28]:
```suggestion
```

File: astro/release-notes-subscribe.md (hunk 13 above)
Comment (jwitz) [reply astro/release-notes-subscribe.md:27]: I take it we don't need the markdown style link in the command itself?

**Review by jwitz — APPROVED** [review]
Congrats on your first (content) contribution @katiebrady-astro ! Just added a few suggestions to clean up the doc, and also a question or two about the commands we ask people to run. Once these suggestions are addressed, this is good to mere.

File: astro/release-notes-subscribe.md (hunk 6 above)
Comment (katiebrady-astro) [reply astro/release-notes-subscribe.md:15]: Yeah that edit works! You can _technically_ view them in Safari, it just doesn't render XML files with proper syntax highlighting and can't natively read them.

File: astro/release-notes-subscribe.md [hunk 15]
```diff
@@ -0,0 +1,27 @@
...
+
+To add a feed to a public channel, the Slack Primary Owner role might be required. To learn more about adding RSS feeds to Slack, see [Add RSS feeds to Slack](https://slack.com/help/articles/218688467-Add-RSS-feeds-to-Slack).
+
+Run each of the following commands to subscribe to different Astro release notes:
+
+- Astro product release notes: `/feed subscribe https://docs.astronomer.io/astro-release-notes.xml`
+
+- Astro Runtime release notes: `/feed subscribe https://docs.astronomer.io/astro-runtime-release-notes.xml`
+
+- Astro CLI release notes:  `/feed subscribe https://docs.astronomer.io/astro-cli-release-notes.xml`
```
Comment (jwitz) [inline astro/release-notes-subscribe.md:26]:
```suggestion
- Astro product release notes: `/feed subscribe https://docs.astronomer.io/astro-release-notes.xml`
- Astro Runtime release notes: `/feed subscribe https://docs.astronomer.io/astro-runtime-release-notes.xml`
- Astro CLI release notes:  `/feed subscribe https://docs.astronomer.io/astro-cli-release-notes.xml`
```

File: astro/release-notes-subscribe.md [hunk 16]
```diff
@@ -0,0 +1,25 @@
...
+
+To have RSS feed updates sent to your email, use a browser RSS feed reader extension such as the [Chrome RSS Feed Reader](https://chrome.google.com/webstore/detail/rss-feed-reader/pnjaodmkngahhkoihejjehlcdlnohgmp?hl=en).
+
+_Note: Safari does not natively support viewing or configuring RSS feeds on Desktop or Mobile._
+
+## Subscribe to Slack updates
+
+To add a feed to a public channel, the Slack Primary Owner role might be required. To learn more about adding RSS feeds to Slack, see [Add RSS feeds to Slack](https://slack.com/help/articles/218688467-Add-RSS-feeds-to-Slack).
+
+Run each of the following commands to subscribe to different Astro release notes:
```
Comment (jwitz) [inline astro/release-notes-subscribe.md:20]:
```suggestion
Run the following commands to subscribe to different Astro release notes:
```

File: astro/release-notes-subscribe.md [hunk 17]
```diff
@@ -1,28 +1,27 @@
...
-## Get Updates on Slack
+## Subscribe to Slack updates
 
-Add the **Astro Release Notes** feed to a Slack Channel with the command  
-/feed subscribe [https://docs.astronomer.io/astro-release-notes.xml](https://docs.astronomer.io/astro-release-notes.xml)
+To add a feed to a public channel, the Slack Primary Owner role might be required. To learn more about adding RSS feeds to Slack, see [Add RSS feeds to Slack](https://slack.com/help/articles/218688467-Add-RSS-feeds-to-Slack).
 
-Add the **Astro Runtime Release Notes** feed to a Slack Channel with the command  
-/feed subscribe [https://docs.astronomer.io/astro-runtime-release-notes.xml](https://docs.astronomer.io/astro-runtime-release-notes.xml)
+Run each of the following commands to subscribe to different Astro release notes:
```
Comment (kwcanuck) [inline astro/release-notes-subscribe.md]: Run one of the following Slack slash commands to subscribe to different Astro release notes:

**Pull request merged**